nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/test/java/org/apache/nifi/processors/azure/storage/ITPutAzureDataLakeStorage.java

```diff
@@ -253,6 +300,14 @@ private void assertFlowFile(String directory, String fileName, byte[] fileData)
         flowFile.assertAttributeEquals("azure.length", Integer.toString(fileData.length));
     }
 
+    private void assertSimpleFlowFile(byte[] fileData) throws Exception {
```

Comment:
I could be called from `assertFlowFile` because the first section of that method is the same.